<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ACCoupling">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="ACCouplingLabel">
       <property name="text">
        <string>AC Coupling</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ACCouplingValue">
       <item>
        <widget class="QPushButton" name="acCouplingYes_TA">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="acCouplingNo_TA">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Gain">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="GainLabel">
       <property name="text">
        <string>Gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gainValue">
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
    <zorder>gainValue</zorder>
    <zorder>GainLabel</zorder>
   </widget>
   <widget class="QLabel" name="AdvancedSettings_TA">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>20</y>
      <width>165</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Advanced Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="HF2TA">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>40</y>
      <width>247</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trans-Impedance Amplifier (HF2TA)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pageUp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ï¼œ--</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
